utils/postActions.js

```diff
@@ -10,15 +10,16 @@ const Axios = axios.create({
 });
 
 
-export async function submitNewPost(text, location, picUrl, setPosts, setNewPost, setError){
+export async function submitNewPost(user,text, location, picUrl, setPosts, setNewPost, setError){

```

Comment:
pass in user object in function parameters and use it to populate user data in new post.